src/world/partners.c

```diff
@@ -377,13 +377,13 @@ void switch_to_partner(s32 arg0) {
 
 void partner_init_after_battle(s32 arg0) {
     PlayerStatus* playerStatus = &gPlayerStatus;
-    Temp8010EBB0* temp8010EBB0 = &D_8010EBB0;
+    PartnerActionStatus* actionStatus = &gPartnerActionStatus;
 
     if (D_8010CFD8 != arg0) {
         D_8010CFE0 = 1;
         D_8010CFE4 = arg0;
-        temp8010EBB0->unk_00 = 0;
-        temp8010EBB0->unk_01 = 0;
+        actionStatus->actionState.b[0] = ACTION_STATE_IDLE;
+        actionStatus->actionState.b[1] = 0;
```

Comment:
What's the rationale for using a constant for b[0] but not b[1]

Reply:
Well we don't know what b[1] represents, but we also don't know what b[0] represents so this was a mistake. I changed it back to 0.

(we do know that b[0] being 0 means idle, but that specific enum shouldn't be used here since partner action states are different)

Reply:
Thought so, thanks for the fix